02_bubble/main.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func bubble(s []int) []int {
+	cnt := len(s)
+	for i := 0; i < cnt; i++ {
+		for j := i + 1; j < cnt; j++ {
+			if s[i] > s[j] {
+				tmp := s[i]
+				s[i] = s[j]
+				s[j] = tmp
```

Comment:
This can be done in one liner s[i], s[j] = s[j], s[i]

Reply:
Done